### PR TITLE
feat(CLI): envd env describe expose info

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* close https://github.com/tensorchord/envd/issues/782

### CODE REVIEW

**Comment by kemingy** [comment]
image

* default sshd port is also displayed

File: pkg/app/image_describe.go [hunk 1]
```diff
@@ -50,6 +48,6 @@ func getImageDependency(clicontext *cli.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to list dependencies")
 	}
-	renderDependencies(dep, os.Stdout)
```
Comment (gaocegege) [inline pkg/app/image_describe.go:53]: It is for test purposes, should we keep it?

Or it is hard to test the feature in the future.

Comment (kemingy) [reply pkg/app/image_describe.go:53]: Do you mean writing to `stdout` is for test purposes?

Even now, testing this feature is problematic unless we support output JSON.

Comment (aseaday) [reply pkg/app/image_describe.go:53]: I guess gaoce's meaning is that we shoud pass the fd currently?

Comment (gaocegege) [reply pkg/app/image_describe.go:53]: We may write UT for `renderDependencies`

Comment (kemingy) [reply pkg/app/image_describe.go:53]: Fixed. PTAL.

Comment (aseaday) [reply pkg/app/image_describe.go:53]: LGTM

**Review by aseaday — APPROVED** [review]
LGTM

**Review by gaocegege — APPROVED** [review]
LGTM :+1:
/lgtm

**Pull request merged**